Tasks/Gulp/gulptask.js

```diff
@@ -1,24 +1,34 @@
 var path = require('path');
 var tl = require('vso-task-lib');
 
-var gt = new tl.ToolRunner(tl.which('gulp', true));
+var nt = new tl.ToolRunner(tl.which('node', true));
 
-// optional - no tasks will concat nothing
-gt.arg(tl.getDelimitedInput('targets', ' ', false));
-
-gt.arg('--gulpfile');
 var gulpFile = tl.getPathInput('gulpFile', true);
-gt.arg(gulpFile);
-
-gt.arg(tl.getDelimitedInput('arguments', ' ', false));
-
 var cwd = tl.getInput('cwd', false);
 if (!cwd) {
 	cwd = path.dirname(gulpFile);
 }
+
+var gulpjs = tl.getInput('gulpjs', false, true);
+if (!gulpjs) {
+	gulpjs = path.resolve(path.join(cwd, 'node_modules/gulp/bin/gulp.js'));
+	tl.checkPath(gulpjs, 'gulp.js');
```

Comment:
Test it not existing.  Until I fix the other issue, it will likely error but continue on?